spring-cloud-starter-stream-common-gemfire/src/main/java/org/springframework/cloud/stream/app/gemfire/config/GemfireClientCacheConfiguration.java

```diff
@@ -22,11 +22,15 @@
 
 import org.springframework.boot.context.properties.EnableConfigurationProperties;
 import org.springframework.context.annotation.Bean;
+import org.springframework.core.io.Resource;
 import org.springframework.data.gemfire.client.ClientCacheFactoryBean;
 import org.springframework.data.gemfire.util.PropertiesBuilder;
 import org.springframework.util.FileCopyUtils;
 import org.springframework.util.StringUtils;
 
+import static org.springframework.cloud.stream.app.gemfire.config.GemfireSslProperties.LOCAL_KEYSTORE_FILE_NAME;
+import static org.springframework.cloud.stream.app.gemfire.config.GemfireSslProperties.LOCAL_TRUSTSTORE_FILE_NAME;
```

Comment:
Our policy do not have `static` imports in the `main` code.

I think it's not critical, but still: we develop here Spring.